packages/volto/src/components/manage/Blocks/Block/StyleWrapper.jsx

```diff
@@ -19,7 +19,13 @@ const StyleWrapper = (props) => {
     classNames,
   });
 
-  style = buildStyleObjectFromData(data);
+  style = buildStyleObjectFromData(
+    data,
+    '',
+    // If we are rendering blocks inside a container, then pass also the data from the container
+    // This is needed in order to calculate properly the styles for the blocks inside the container
+    isContainer && content.blocks ? content : {},
```

Comment:
I'm not sure I understand what you mean by container here.

The comment says it's passing the data from the container, which I assumed was a block, but the code is passing the full content item, isn't it?

Reply:
@davisagli tests fixed.

Reply:
I agree :(